### PR TITLE
Add v1.3.0 master copies support for VOLTA

### PR DESCRIPTION
**Pull request by Uxio0**
Add v1.3.0 master copies support for VOLTA

@gnosis/safe-services

### CODE REVIEW

File: safe_transaction_service/history/management/commands/setup_service.py [hunk 1]
```diff
@@ -114,8 +114,8 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ('0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F', 6399212, '1.1.1'),
     ],
     EthereumNetwork.VOLTA: [
-        # ('0x3E5c63644E683549055b9Be8653de26E0B4CD36E', 0, '1.3.0+L2'),
-        # ('0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552', 0, '1.3.0'),
+        ('0x3E5c63644E683549055b9Be8653de26E0B4CD36E', 11828662, '1.3.0+L2'),
```
Comment (fmrsabino) [inline safe_transaction_service/history/management/commands/setup_service.py:117]: I think the block number here should be `11942450`

See: https://volta-explorer.energyweb.org/tx/0xb2de05e31b2f1397252d480ee1cbc1ea4f505aa65dce0e8073f548c5e393bb03/internal-transactions

File: safe_transaction_service/history/management/commands/setup_service.py [hunk 2]
```diff
@@ -114,8 +114,8 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
...
+        ('0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552', 11942450, '1.3.0'),
```
Comment (fmrsabino) [inline safe_transaction_service/history/management/commands/setup_service.py:118]: And for this one it seems that it was created with this tx: https://volta-explorer.energyweb.org/tx/0xa8eae8dd93a4dffdcb28d39800cad926e58a546ddfbfaf1cb35c5fc6c708f7b0/internal-transactions

So block number would be `11942451`

**Review by fmrsabino — CHANGES_REQUESTED** [review]

File: safe_transaction_service/history/management/commands/setup_service.py (hunk 2 above)
Comment (Uxio0) [reply safe_transaction_service/history/management/commands/setup_service.py:118]: Sure!

**Comment by Uxio0** [comment]
Updated

**Review by fmrsabino — APPROVED** [review]
🚀

**Review by rmeissner — APPROVED** [review]

**Pull request merged**